### ISSUE TITLE
Dynamic and hybrid trajectory set for covernet

### ISSUE THREAD

**Issue by WenchaoDing** [nutonomy/nuscenes-devkit#386]
Hi there!

Thanks for your amazing work! I used the given trajectory sets (eps=2m, 4m, 8m) and have successfully reproduced your results in your covernet paper. I am wondering whether you can further provide the dynamic and hybrid trajectory sets since they may lead to better results on nuscenes as indicated by your paper. This will be very useful for my benchmarking. Thanks a lot!

**Comment by freddyaboulton** [comment]
Hi @WenchaoDing Sorry for the late reply but thanks for your question!

We don't have plans to release the dynamic or hybrid trajectory sets right now but we will reconsider later in the summer!

**Comment by Vic2901** [comment]
Hi Mr. Ding,
I also train the model from scratch by using the mini-dataset, but I got a problem, that is the validation loss basically does not go down, or changes pretty little. I was wondering how you deal with the NaN entries in the agent state vector, because I directly force it as a very small positive number, is it reasonable? Thx:)

**Comment by XHwind** [comment]
Hi ~I am also trying to reproduce the result of CoverNet. And currently I find that the loss function couldn't converge and just oscillate. I am wondering would you by any chance consider releasing the training settings (like learning rate, batch size, etc)? Thank you!

**Comment by srikanthmalla** [comment]
Hi @freddyaboulton , Does the current nuscenes api support dynamic or hybrid trajectory sets for inference?
